kalite/distributed/management/commands/persist.py

```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand, CommandError
+import os
+import json
+from sqlitedict import SqliteDict
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        print("Converting...")
+        self.convert('data/khan/contents.json', 'data/khan/contents.sqlite')
+        self.convert('data/khan/exercises.json', 'data/khan/exercises.sqlite')
+
+    def convert(self, jsonurl, sqliteurl):
+        url = os.path.join(os.environ['KALITE_DIR'], jsonurl)
+        try:
+            jsonfile = open(url)
+            items = json.load(jsonfile)
+            kalitedict = SqliteDict(
+                os.path.join(os.environ['KALITE_DIR'], sqliteurl))
+        except IOError, e:
+            print(e)

```

Comment:
Rather than using print statements, can you try using the logging module? You can import it like this:

```python
from django.conf import settings
logging = settings.LOG

<...>

logging.info("Converting")
logging.error(e)
```